configs/Config.cmake.in

```diff
@@ -7,7 +7,22 @@ include(CMakeFindDependencyMacro)
 # the place of this comment. Make sure they match up with the
 # find_package calls in your package's CMakeLists.txt file
 
+
+
+# Figure out whether or not this dependency is an installed package or
+# in repo form
+
+if (EXISTS ${CMAKE_SOURCE_DIR}/@PROJECT_NAME@)
```

Comment:
This `if` statement, do you see it as a permanent solution, or is there a hope to unify the two cases into one command (`add_library` hopefully?)

Reply:
OK, after a second look I realise I need to be educated about what `Config.cmake.in` 😅